zerver/views/messages.py

```diff
@@ -863,6 +866,40 @@ def json_update_message(request, user_profile):
     return update_message_backend(request, user_profile)
 
 @has_request_variables
+def highlight_message_differences(message_history, sender_email):
```

Comment:
Since this function doesn't have any request variables, you don't need the decorator here.